tools/tests/stepper/test.js

```diff
@@ -39,21 +39,21 @@ describe('Stepper Component', () => {
 
     it('Should have correct selected option', async () => {
         const selected = STEPPER_OPTIONS_SELECTED.find(option => option.selected);
-        await loadStepper(STEPPER_OPTIONS_SELECTED);
+        await loadStepper(STEPPER_OPTIONS_SELECTED, selected.name);
         const stepper = document.querySelector('gameface-stepper');
         assert.equal(stepper.value, selected.name);
     });
 
     it('Should have correct selected index', async () => {
-        const selected = STEPPER_OPTIONS_SELECTED.findIndex(option => option.selected);
-        await loadStepper(STEPPER_OPTIONS_SELECTED);
+        const selectedIndex = STEPPER_OPTIONS_SELECTED.findIndex(option => option.selected);
+        await loadStepper(STEPPER_OPTIONS_SELECTED, STEPPER_OPTIONS_SELECTED[selectedIndex].name);
         const stepper = document.querySelector('gameface-stepper');
-        assert.equal(stepper.selectedIndex, selected);
+        assert.equal(stepper.selectedIndex, selectedIndex);

```

Comment:
Am I wrong or all the tests are missing an error message in the `assert`?

Reply:
Well, we can omit the error messages here because if they fail they will display a default message that something is not equal. For example, you may receive the following error for the `assert.equal(stepper.value, STEPPER_OPTIONS_SELECTED[2].name);`
```
[Fail] Should not select next option if last option is selected
AssertError: 'Option 1' === 'Option 2'
```
that is descriptive enough. You know the test name and the received values for the assert.

However, this assert checks something complex `assert.equal(stepper[attrName], window[templateName][attrName], `The stepper has a different ${attrName}. Received ${stepper[attrName]}. Expected ${window[templateName][attrName]}`)`
and you need to have a custom message to know which attribute has failed :D

Reply:
Agreed!
Maybe we have over-delivered on the messages in the tests.
This is fast, simple and is traceable!